Ds3/Helpers/Strategies/ChunkStrategies/ReadStreamChunkStrategy.cs

```diff
@@ -0,0 +1,173 @@
+﻿/*
+ * ******************************************************************************
+ *   Copyright 2014-2016 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using Ds3.Calls;
+using Ds3.Models;
+
+namespace Ds3.Helpers.Strategies.ChunkStrategies
+{
+    /// <summary>
+    ///  The ReadStreamChunkStrategy will get the available job chunks, allocate those chunks and send the blobs in order
```

Comment:
This comment needs to be updated since this strategy is not actually allocating chunks, and it is not sending them, but rather receiving chunks.